hummingbot/connector/exchange/bittrex/bittrex_api_order_book_data_source.py

```diff
@@ -1,244 +1,186 @@
+    def _channel_originating_message(self, event_message) -> str:
+        channel = self._trade_messages_queue_key
+        if event_message["type"] == "orderBook":
+            channel = self._diff_messages_queue_key
+        return channel
+
+    async def _parse_trade_message(self, raw_message: Dict[str, Any], message_queue: asyncio.Queue):
+        trading_pair = await self._connector.trading_pair_associated_to_exchange_symbol(symbol=raw_message["marketSymbol"])
+        for data in raw_message["deltas"]:
+            trade_message: OrderBookMessage = self.trade_message_from_exchange(
+                msg=data,
+                metadata={"trading_pair": trading_pair, "sequence": raw_message["sequence"]}
+            )
+            message_queue.put_nowait(trade_message)
+
+    async def _parse_order_book_diff_message(self, raw_message: Dict[str, Any], message_queue: asyncio.Queue):
+        trading_pair = await self._connector.trading_pair_associated_to_exchange_symbol(symbol=raw_message["marketSymbol"])
+        order_book_message: OrderBookMessage = self.diff_message_from_exchange(raw_message, time.time(), {"trading_pair": trading_pair})
+        message_queue.put_nowait(order_book_message)
+
+    def snapshot_message_from_exchange(self, msg: Dict[str, any],
+                                       timestamp: float,
+                                       metadata: Optional[Dict] = None) -> OrderBookMessage:
+        if metadata:
+            msg.update(metadata)
+        bids, asks = msg["bid"], msg["ask"]
+        bids = [(bid["rate"], bid["quantity"]) for bid in bids]
+        asks = [(ask["rate"], ask["quantity"]) for ask in asks]
+        return OrderBookMessage(OrderBookMessageType.SNAPSHOT, {
+            "trading_pair": msg["trading_pair"],
+            "update_id": int(timestamp),
+            "bids": bids,
+            "asks": asks
+        }, timestamp=timestamp)
+
+    def trade_message_from_exchange(self, msg: Dict[str, Any],
+                                    timestamp: Optional[float] = None,
+                                    metadata: Optional[Dict] = None) -> OrderBookMessage:
+        if metadata:
+            msg.update(metadata)
+        return OrderBookMessage(
+            OrderBookMessageType.TRADE, {
+                "trading_pair": msg["trading_pair"],
+                "trade_type": float(TradeType.BUY.value) if msg["takerSide"] == "BUY" else float(TradeType.SELL.value),
+                "trade_id": msg["id"],
+                "update_id": msg["sequence"],
+                "price": msg["rate"],
+                "amount": msg["quantity"]
+            }, timestamp=_get_timestamp(msg["executedAt"]))
+
+    def diff_message_from_exchange(self, msg: Dict[str, any],
+                                   timestamp: Optional[float] = None,
+                                   metadata: Optional[Dict] = None):
+        if metadata:
+            msg.update(metadata)
+        bids, asks = msg["bidDeltas"], msg["askDeltas"]
```

Comment:
This line does not make sense if in the next lines the variables are being overwritten

Reply:
Removed the extra line